.codespellignore

```diff
@@ -0,0 +1,5 @@
+aks
+witht
+geting
+ot
+intepreted
```

Comment:
sad how that's an actual typo in the calico manifests... maybe we can fix it in the calico repo and then get rid of this later...

Reply:
Yeah, I'll fix it upstream :)

Reply:
Opened https://github.com/projectcalico/calico/pull/6067

Reply:
https://github.com/projectcalico/calico/pull/6067#event-6618246089 got merged. So at some point it will be fixed.